pyspecdata/plot_funcs/pcolormesh.py

```diff
@@ -10,12 +10,15 @@ def pcolormesh(
     ax2=None,
     ax=None,
     scale_independently=False,
+    vmin=None,
+    vmax=None,
```

Comment:
these are good to have, but not currently used.

I think that if these are set, you (1) throw an error if `scale_independently` or `force_balanced_cmap` are set, b/c that doesn't make sene and (2) overwrite the values of overall_max and overall_min with these values.

Reply:
I don't see why I need to have vmin and vmax overwrite overall_max and _min, I think they should just be whatever they are manually set to and then appended to set_clim for modifying the colorbar

Reply:
this way may have worked but its probably better to only make one call to `set_clim` at the end for adjusting the colorbar